frontend/src/components/hub/ContactAmbassadorButton.js

```diff
@@ -0,0 +1,58 @@
+import { Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React, { useContext } from "react";
+import { redirect } from "../../../public/lib/apiOperations";
+import { startPrivateChat } from "../../../public/lib/messagingOperations";
+import Router from "next/router";
+import UserContext from "../context/UserContext";
+import getTexts from "../../../public/texts/texts";
+import Cookies from "universal-cookie";
+import ContactCreatorButtonInfo from "../communication/contactcreator/ContactCreatorButtonInfo";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    zIndex: 1,
+    position: "fixed",
+    bottom: 0,
+    right: "200px",
```

Comment:
The positioning looks a bit weird on smaller screen sizes. Perhaps we could reuse the code from the contact project creator button and make it just not be all the way at the bottom when you're scrolled down all the way.
And we need a solution for mobile @tobiasrehm

Reply:
I changed it to 10% so it deals better with different screen sizes :-)